theme/svg.go

```diff
@@ -0,0 +1,111 @@
+package theme
+
+import (
+	"encoding/hex"
+	"encoding/xml"
+	"fmt"
+	"image/color"
+	"io"
+	"io/ioutil"
+)
+
+// SVG holds the unmarshaled XML from a Scalable Vector Graphic
+type SVG struct {
```

Comment:
Can we keep this class, and it's fields, private at the moment? I don't think there is a need to add it to the public API?

Reply:
Agreed.  Being new to Go, I don't know if I did this correctly.  I made the struct private, but I had to keep the fields capitalized since they have annotations (I think, the IDE complained when I made the fields lowercase).  Let me know if further work needs to be done on this.

Reply:
Quite right, sorry I missed the annotations requirement. keeping the name un-exported will remove it from our public API.